spec/osut_tests_spec.rb

```diff
@@ -2937,7 +2937,7 @@ module M
     vtx << mini_attic_vtx[2]
     vtx << mini_attic_vtx[1]
     vtx << mini_attic_vtx[0]
-    vtx << mini_attic_vtx[3]
+    vtx << mini_attic_vtx[3] # ?
```

Comment:
The failed RSpec deals with _holes_ cut in (larger) surfaces, to accommodate either other (smaller) surfaces or shafts (e.g. skylight wells). Roughly half of the comments/illustrations of this merged [PR](https://github.com/rd2/osut/pull/13) describe encountered pitfalls/workarounds for a handful of scenarios. The solution rests on properly winding [extra](https://github.com/rd2/osut/pull/13#discussion_r1276124965) vertices (of the larger surface) around a new _hole_.

The RSpec passes for OpenStudio versions 3.0.0 to 3.6.1, yet fails for 3.7.0-rc1 (/spec/osut_tests_spec.rb:2978):

```
Failure/Error: expect(attic.volume).to be_within(TOL).of(720.19) unless v < 350
       expected 798.4139451428566 to be within 0.01 of 720.19
```  

_Note that a fix to OpenStudio volume calculations was introduced in v3.5.1. Recent releases correctly calculate/report space/zone volume (in sync with EnergyPlus). For this reason, OpenStudio version checks are found in the RSpec (e.g. `unless v < 350`)._

OpenStudio v3.7.0-rc1 reports the following messages:  

```
[utilities.Polyhedron] <0> Polyhedron is not enclosed in original testing. Trying to add missing colinear points.
[utilities.Polyhedron] <0> Polyhedron is not enclosed.
[openstudio.model.Space] <0> Object of type 'OS:Space' and named 'Core_ZN' is not enclosed, there are 2 edges that aren't used exactly twice. Falling back to ceilingHeight * floorArea. Volume calculation will be potentially inaccurate.
[utilities.Polyhedron] <0> Polyhedron is not enclosed in original testing. Trying to add missing colinear points.
[utilities.Polyhedron] <0> Polyhedron is not enclosed.
[openstudio.model.Space] <0> Object of type 'OS:Space' and named 'Attic' is not enclosed, there are 1 edges that aren't used exactly twice. Falling back to ceilingHeight * floorArea. Volume calculation will be potentially inaccurate.
```  

... and reverts to pre-3.5.1 volume calculations. However, a forward translated IDF (v23.2.0) of the same modified model (`files/osms/out/miniX.idf`) reports the right volume (720.19 m3 for the attic). This remains consistent with older EnergyPlus versions (tested for v9.5.0 up to v23.2.0).

<img width="525" alt="mini1" src="https://github.com/rd2/osut/assets/98104985/326adea5-5d7f-4025-9e9a-dd6e0ded4a43">

There are no EnergyPlus-reported warnings or errors with IDFs generated using the solution, regardless of the version tested. OpenStudio v3.5.1 and v3.6.1 (with fixed volume calculations) did not report this as an issue either.

This seems to stem from revised checks discussed [here](https://github.com/NREL/OpenStudio/pull/4843).